pkg/features/kube_features.go

```diff
@@ -135,6 +135,7 @@ const (
 
 	// owner: @k82cn
 	// beta: v1.12
+	// GA: v1.17
```

Comment:
from GA feature, we just remove it :)

Reply:
and for this feature, it's also to remove some predicates about node condition.

Reply:
Please take another look, thanks!

Reply:
No - we don't remove feature gates for GA (at least not immediately - otherwise if someone is setting up feature gate, we will break them).

Reply:
we *do* lock to true when promoting to GA (`LockToDefault: true` in the feature gate setup), and remove code that checks the gate

Reply:
Addressed

Reply:
@liggitt , @wojtek-t , when should remove the function code feature? Only keep feature gate for one release :)

And good to know we did not remove it immediately :)

Reply:
one release. announce graduation to GA and feature gate deprecation in 1.17, then remove in 1.18

Reply:
got that, thanks very much :)